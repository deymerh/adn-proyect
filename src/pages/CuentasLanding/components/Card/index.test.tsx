import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import { OutlinedCard } from './index';

describe('Pruenas en el componente <OutlinedCard/>', () => {
  const props = {
    idCuenta: 123,
    tipoCuenta: 'ahorro',
    estado: 1,
    valorDisponible: 320.5,
  };
  const wrapper = render(
    <BrowserRouter>
      <OutlinedCard {...props} />
    </BrowserRouter>,
  );
  test('Debe renderizar el componente', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
