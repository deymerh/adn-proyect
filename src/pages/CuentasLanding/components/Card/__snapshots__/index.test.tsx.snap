// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruenas en el componente <OutlinedCard/> Debe renderizar el componente 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiPaper-root MuiCard-root makeStyles-root-1 MuiPaper-outlined MuiPaper-rounded"
      >
        <div
          class="MuiCardContent-root"
        >
          <p
            class="MuiTypography-root makeStyles-title-3 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
          >
            Dia 
            19
             del Mes 
            8
          </p>
          <h2
            class="MuiTypography-root MuiTypography-h5"
          >
            320.5
          </h2>
          <p
            class="MuiTypography-root makeStyles-pos-4 MuiTypography-body1 MuiTypography-colorTextSecondary"
          >
            Saldo disponible
          </p>
          <p
            class="MuiTypography-root MuiTypography-body2"
          >
            Estado de cuenta: 
            Activa
            <br />
            Tipo de cuenta: 
            ahorro
          </p>
        </div>
        <div
          class="MuiCardActions-root MuiCardActions-spacing"
        >
          <a
            href="/cuenta"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textSizeSmall MuiButton-sizeSmall"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                VER DETALLE
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </a>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiPaper-root MuiCard-root makeStyles-root-1 MuiPaper-outlined MuiPaper-rounded"
    >
      <div
        class="MuiCardContent-root"
      >
        <p
          class="MuiTypography-root makeStyles-title-3 MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
        >
          Dia 
          19
           del Mes 
          8
        </p>
        <h2
          class="MuiTypography-root MuiTypography-h5"
        >
          320.5
        </h2>
        <p
          class="MuiTypography-root makeStyles-pos-4 MuiTypography-body1 MuiTypography-colorTextSecondary"
        >
          Saldo disponible
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2"
        >
          Estado de cuenta: 
          Activa
          <br />
          Tipo de cuenta: 
          ahorro
        </p>
      </div>
      <div
        class="MuiCardActions-root MuiCardActions-spacing"
      >
        <a
          href="/cuenta"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textSizeSmall MuiButton-sizeSmall"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiButton-label"
            >
              VER DETALLE
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </a>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
